src/main/kotlin/org/jitsi/nlj/rtcp/KeyframeRequester.kt

```diff
@@ -92,14 +106,16 @@ class KeyframeRequester : TransformerNode("Keyframe Requester") {
                 if (forward) {
                     // When we forward a FIR we need to update the seq num.
                     packet.seqNum = firCommandSequenceNumber.incrementAndGet()
+                    numFirsForwarded++
                 }
+                if (!canSend) numFirsDropped++
             }
             // This is now possible, but the compiler doesn't know it.
             else -> throw IllegalStateException("Packet is neither PLI nor FIR")
         }
 
         if (!forward && canSend) {
-            requestKeyframe(sourceSsrc, now)
+            doRequestKeyframe(sourceSsrc)
```

Comment:
why scrap the current time from the params? typo maybe?

Reply:
`doRequestrKeyframe` does not perform the throttling check and thus doesn't need the time (it doesn't take it as a param).